container/examples/compound-container/index.html

```diff
@@ -0,0 +1,57 @@
+<!-- .js => webcomponent -->
+<!-- .html => iFrame -->
+
+<!-- npm install --save-prod @luigi-project/container --prefix . -->
+
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <title>Luigi Compound Container</title>
+  </head>
+
+  <body>
+    <luigi-compound-container
+      viewURL="./myCompoundWebComponent1.js"
+      id = "compound1"
+      webcomponent="true"
+      context='{"label":"some extra content 1"}'
+    ></luigi-compound-container>
+    
+    <luigi-compound-container
+      viewURL="./myCompoundWebComponent2.js"
+      id = "compound2"
+      webcomponent="true"
+      context='{"label":"some extra content 2"}'
+    ></luigi-compound-container>
+  </body>
+
+  <script type="module">
+    import './bundle.js';
+    
+    document.getElementById('compound1').compoundConfig = {
```

Comment:
Please follow the example setup in here: 
https://github.com/SAP/luigi/blob/2d8c469236ccedbbdc0e785b21bad1fe0508cc90/container/test-app/index.html#L420-L509

That was is expected of in here. 

The value of compound container is best seen in a grid layout. The current config you have does not make use of it. 

Take the config example pointed above as is, then play around with it until you have a working version of it. 

You do not have to use the same exact web component .js files as in there. Just use the simple ones you created. 

Also, please add one more web component. Currently you have 3 files .js only. Create another one to make it 4.

The end result should ideally be a compound container structure built with 4 web components. It should be visually appearing in a grid format at the end. You are not doing any CSS for grid here, as that is already handled by the compoundConfig.renderer configuration. 

Take a look at this part of the documentation to understand what these compound attributes mean:
https://docs.luigi-project.io/docs/navigation-parameters-reference/?section=compound